spec/requests/api/v1/events/events_spec.rb

```diff
@@ -87,4 +83,73 @@
       expect(response).to have_http_status(:unprocessable_entity)
     end
   end
+
+  describe 'PATCH /v1/events/:id' do
+    it 'updates the events attributes' do
+      event = create(:event, name: 'Old Name')
+      new_name = 'New Name'
+      event_params = {
```

Comment:
I said this to @kms617 too, but you can use FactoryGirl's [attributes_for](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#using-factories) instead of building the params manually.